### ISSUE TITLE
Why are sections in an unneccessary submenu now?

### ISSUE THREAD

**Issue by IdrisQe** [modrinth/code#2881]
Not a bug, but just questioning the design decisions here... There is plenty of space at the top of the screen, it's not like people will be downloading Java Minecraft mods on their phones or anything. Why put categories in a submenu like this? At first I thought it was broken and outright missing. This is just worse ease of use for no apparent reason?

![Image](https://github.com/user-attachments/assets/c89ab55f-2717-4238-9bfe-82d525943ca5)

It's even worse since it always just says "Discover content", so if you aren't sure what the icons that show beside that text mean, it might be confusing as to what section you're browsing, particularily for less knowledeable users (i.e. those who don't understand the differences between resource packs and datapacks, or mods and modpacks, which there are an unfortunately large number of.)

**Comment by Franco227** [comment]
Agreed. As someone who uses modrinth on a daily basis, this extra click is just annoying and has no upside.

**Comment by izmystic** [comment]
In the future it will just be a button with a universal search page, in the meantime you can go to https://modrinth.com/flags and enable `projectTypesPrimaryNav`

**Comment by Prospector** [comment]
it's also not an extra click, it's a hover menu. The only difference is you need to adapt to a different position to move your mouse to, within a few days you should be used to it

**Comment by Felix14-v2** [comment]
> it's also not an extra click, it's a hover menu

How can I avoid an additional click when I use keyboard to navigate? The thing is that this is still an additional action, it doesn't matter whether this action is a click or not.

**Comment by ajaxnotmeta** [comment]
why not made that system for Low res screen or small screen(eg. Steam deck)? its soo much screen real estate empty for no reason at all, i agree we need this UI for Smaller but NOT for higher one. Didnt we already have backlash some motnh before where you guys add the similar system but with clicks?(now its better as it dont need clicks, but still)

**Comment by Prospector** [comment]
I expect the upcoming search/discover page will deal with your issues with the current navigation
